src/components/WorkOrder/MobileWorkingWorkOrderView.js

```diff
@@ -172,6 +172,7 @@ const MobileWorkingWorkOrderView = ({ workOrderReference }) => {
           files,
           workOrderReference,
           'Closing work order',
+          '',
```

Comment:
what is the reason for adding this?

Reply:
Added a description param, but the other forms dont have the field yet.